plugins/by-name/perfanno/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, ... }:
+
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "perfanno";
+  packPathName = "perfanno.nvim";
+  package = "perfanno-nvim";
+  description = "NeoVim lua plugin that annotates source code with profiling information from perf, LuaJIT, or other profilers.";
+
+  maintainers = [ lib.maintainers.fredeb ];
+
+  settingsExample = {
+    line_highlights = lib.nixvim.literalLua ''require("perfanno.util").make_bg_highlights(nil, "#CC3300", 10)'';
```

Comment:
Sadly, using the wrong one doesn't cause a CI failure, but I believe we need `nestedLiteralLua` here instead of `literalLua`.

You may also wish to switch to using single quotes in your lua, as I don't think it is smart enough to render a `''` string instead of `"`, unless the string includes newlines.

You can confirm how it renders locally, using `nix run .#docs` (which may be a _long_ build on the initial run).

Reply:
Fair. I did kick off the docs build, but it's still going, so I'm still waiting for the result. I'm at 7650/8389 now, so I'll wait for it to finish and validate

Reply:
Yes, it's a shame our docs take so long. I've put effort in the past into re-implementing the docs more efficiently, but sadly none of those efforts ever made it over the line.